frontend/widgets/SwellActiveCampaigns/style.js

```diff
@@ -0,0 +1,30 @@
+import { css } from 'glamor';
+
+const container = css({
+  margin: '0 auto',
+  display: 'flex',
+  flexWrap: 'wrap',
+  width: '90%',
+
+}).toString();
+
+/**
+ * @param {string} backgroundColor backgroundColor
+ * @param {string} color color
+ * @returns {Object}
+ */
+const card = (backgroundColor, color) => css({
```

Comment:
the boxes are not centered correctly and also scale not correct.
the should be always squares + 50% of the width (including padding around)

![image](https://user-images.githubusercontent.com/5811286/74436582-fad33d80-4e66-11ea-8817-418c25a127c0.png)

Reply:
I struggled to create perfect squares with flex styling, so i had to use float and kept the margin between boxes. Not sure if there is a better way to implement this. Please let me know :)

Reply:
now it looks like this on iphone SE 
![image](https://user-images.githubusercontent.com/5811286/74521985-9f628780-4f1a-11ea-95e8-38e5aa651fa0.png)

i will take care of it

Reply:
google is your friend ;) https://codepen.io/kristencodes/pen/BopVyB